schedule/yast/textmode/textmode@svirt-xen.yaml

```diff
@@ -40,6 +40,11 @@ schedule:
   - console/textinfo
   - console/orphaned_packages_check
   - console/consoletest_finish
+conditional_schedule:
+  stop_reboot_timeout:
+    MACHINE:
```

Comment:
please use here `VIRSH_VMM_TYPE  linux` we don't trust much in MACHINE for conditional schedule (might change name in the future), also it is a setting that belong more to job groups, so even if it is less clear that linux corresponds to svirt-xen-pv is the common approach, otherwise if you disagree then it is better to have two files for clarity.

Reply:
good point, I went with `MACHINE` for the straightforwardness of the svirt-xen-pv setting but in the end MACHINE names are not prone to staying the same for ever

Reply:
I restarted the verification runs